platformio.ini

```diff
@@ -30,10 +30,19 @@ default_envs =
 upload_port     = 
 monitor_port    = 
 upload_speed    = 1500000
-
+check           = cppcheck
 build_flags     = -std=gnu++17
                   -Dregister=                       ; Sinister: redefine 'register' so FastLED can use that keyword under C++17
 
+;                 -Wall                             ; Extra useful warnings - uncomment to enable and fix anything you find in OUR code!
+;                 -Wformat=2                        ;   but you'll set many warnings in the libraries, so don't enable this in the build
+;                 -Wformat-truncation               ;   until you're ready to fix them all :-)
+;                 -Wstack-usage=4096           
+;                 -g3
+;                 -Os
+;                 -ffunction-sections
```

Comment:
Consider leaving the -ffoo-secctions and -Os turned on in the default builds. They cost a fraction of a second at link time, but save .text and .data size by discarding library code we don't call. Flash is a precious resource and economizing it worth it.

My tree has these turned on. No observed problems.

Reply:
Done!